packages/sdk-codegen/src/sdkModels.spec.ts

```diff
@@ -732,6 +732,18 @@ describe('sdkModels', () => {
 
   describe('method and type xrefs', () => {
     describe('custom types', () => {
+      it('determines parent type of array types', () => {
```

Comment:
maybe add a test for "parent of hash types" since your fix in sdkModels is

```
propType.instanceOf('ArrayType') || propType.instanceOf('HashType')
```

(or conversely remove the `|| HashType` condition)

Reply:
We have no complex hash types in the model right now, so we need to create a sample spec for that.

Reply:
I didn't add a test for this because unfortunately we do not have any typed `HashType` in the spec. All parameters of type `IDictionary` expect a scalar property value. I added the check because I expect it to work when we do, but perhaps I should remove it so that when we do need it, we'll remember to add a test

Reply:
we've modified `test/openApiRef.json` in the past to add spec schema that we don't already have in Looker: https://github.com/looker-open-source/sdk-codegen/commit/1b64e1b66a234a67f3a71d85f1545776334f1894#diff-90f6053ccc4c15adb9cbbba6833dbfb5a6206c6fb3793fe206f10f71230b73faR29331

So yeah, I'd say either remove the check (perhaps adding a TODO) or add the test

Reply:
added the test